### ISSUE TITLE
eslint 1.10.3 keeps bugging me about incorrect identation (but it is correct)

### ISSUE THREAD

**Issue by zewish** [eslint/eslint#4739]
I'm not sure if I missed some option in .eslintrc.json, but this is what I get with the following test files:

t.js:
```javascript
'use strict';

function A() {
    this.a = function() {
        'test';
        'test';
        return 'a';
    };

    this.b = function() {
        return this.a();
    };

    this.c = {
        aa: function() {
            'test1';
            'test1';
            return 'aa';
        }
        , bb: function() {
            return this.bb();
        }
    };
}

module.exports = new A();
```

.eslintrc.json:
```json
{
    "rules": {
        "indent": [
            1,
            4
        ],
        "quotes": [
            2,
            "single"
        ],
        "linebreak-style": [
            2,
            "unix"
        ],
        "semi": [
            2,
            "always"
        ]
    },
    "env": {
        "es6": false,
        "node": true
    },
    "extends": "eslint:recommended"
}
```

output:
```bash
[wish@wish wish]$ eslint ./t.js 

/home/wish/t.js
  21:13  warning  Expected indentation of 14 space characters but found 12  indent
  22:9   warning  Expected indentation of 10 space characters but found 8   indent

✖ 2 problems (0 errors, 2 warnings)
```

**Comment by ilyavolodin** [comment]
Looks like an issue with multiline indent and comma-first style. @gyandeeps could you take a look? I'm sure it's a duplicate of some existing issue.

**Comment by gyandeeps** [comment]
Confirmed, the issue is happening because of the comma first-style.
Somewhat related https://github.com/eslint/eslint/issues/3456
